packages/teams-js/src/internal/utils.ts

```diff
@@ -338,3 +339,24 @@ export function createTeamsAppLink(params: pages.NavigateToAppParams): string {
   }
   return url.toString();
 }
+
+/**
+ * @hidden
+ * Checks if the Adaptive Card schema version to check for host support.
+ * @param hostAdaptiveCardSchemaVersion Hosts supported Adaptive Card Version in the runtime.
```

Comment:
How about:

Host's supported Adaptive Card version in the runtime

(Added apostrophe to "Host" and de-capitalized "Version")